src/main/java/org/takes/rs/RsWithBody.java

```diff
@@ -51,7 +52,8 @@
 public final class RsWithBody extends RsWrap {
 
     /**
-     * Ctor.
+     * Constructs a {@code RsWithBody} with the specified body that will be
+     * encoded into UTF-8 by default.
```

Comment:
@mkordas Unless I missed a constructor but for me actually it is consistent because the other constructors deal with already serialized content such as byte[], InputStream or URL so I have no way to know how it has been encoded.